llvm/lib/TargetParser/AArch64TargetParser.cpp

```diff
@@ -11,11 +11,13 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "llvm/TargetParser/AArch64TargetParser.h"
+#include "llvm/Support/Error.h"
```

Comment:
Why do we need to include Support/Error now?

These includes shouldn't change too much, the "module" include `AArch64TargetParser.h` will always come first regardless of name order, clang-format should enforce that for you.